src/service/standard-actions-impl.js

```diff
@@ -88,14 +103,62 @@ export class StandardActions {
    */
   handleHide(invocation) {
     const target = dev().assertElement(invocation.target);
+
     this.resources_.mutateElement(target, () => {
       if (target.classList.contains('i-amphtml-element')) {
         target./*OK*/collapse();
       } else {
         toggle(target, false);
+        target.setAttribute('hidden', '');
```

Comment:
I don't think so. Support has been around for a while http://caniuse.com/#feat=hidden All versions of Chrome and Firefox, iOS > 5, IE11, and all versions of Edge support it.